modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/DDMTemplateLocalServiceTest.java

```diff
@@ -215,6 +217,33 @@ public void testDeleteTemplateByExternalReferenceCode() throws Exception {
 			_ddmTemplateLocalService.fetchTemplate(template.getTemplateId()));
 	}
 
+	@Test
+	public void testFetchDDMTemplateByExternalReferenceCodeIncludingAncestors()
```

Comment:
Could you rename it to: testFetchDDMTemplateByExternalReferenceCodeIncluding**AncestorTemplates**